### PR TITLE
--verbose switch for function create

### PR DESCRIPTION
**Pull request by glyn**
Fixes https://github.com/projectriff/riff/issues/623

### CODE REVIEW

**Comment by trisberg** [comment]
so, first attempt testing and I get an error

```
Waiting for LatestCreatedRevisionName:..................Error: function create failed: RevisionMissing: Configuration "upper" does not have any ready Revision.
```

maybe we could at that point show some info regarding that error, why did the kservice/configuration not create a revision? 

checking the `configuration.serving.knative.dev/upper` status: 
```
Status:
  Conditions:
    Last Transition Time:  2018-08-16T15:13:13Z
    Message:               Revision creation failed with message: "Internal error occurred: admission webhook \"webhook.build.knative.dev\" denied the request: mutation failed: serviceaccounts \"riff-build\" not found".
    Reason:                RevisionFailed
    Status:                False
    Type:                  Ready
```

so, it was user error, I didn't run `riff namespace init` with the secret

**Comment by glyn** [comment]
@trisberg Fixed with an extra commit. The output in the above case now looks like this:
```
Waiting for LatestCreatedRevisionName:Error: function create failed: RevisionMissing: Configuration "square" does not have any ready Revision.; Revision creation failed with message: "Internal error occurred: admission webhook \"webhook.build.knative.dev\" denied the request: mutation failed: serviceaccounts \"riff-build\" not found".
```

File: cmd/commands/shared.go [hunk 1]
```diff
@@ -27,4 +27,5 @@ const (
 or 'secretKeyRef' to select a key from a Secret. The following formats are supported:
   --env-from configMapKeyRef:{config-map-name}:{key-to-select}
   --env-from secretKeyRef:{secret-name}:{key-to-select}`
+	verboseUsage = "print details of command progress"
```
Comment (trisberg) [inline cmd/commands/shared.go]: should this '=' line up with the ones above? Seems that is the style used here and in other places. Not sure what the typical Go formatting for this is.

**Comment by trisberg** [comment]
LGTM

This works nicely. I was able to rebase my `function build` command on this PR to add the `--verbose` option, works great.

File: cmd/commands/shared.go (hunk 1 above)
Comment (glyn) [reply cmd/commands/shared.go]: @trisberg It's working as designed! There is one standard formatting rule, defined by `go fmt`, but sometimes it's a bit too subtle and looks wrong. See [this comment](https://github.com/golang/go/issues/7819#issuecomment-66093841)  about a similar issue to get a flavour...

Comment (glyn) [reply cmd/commands/shared.go]: @trisberg It's working as designed! There is one standard formatting rule, defined by `go fmt`, but sometimes it's a bit too subtle and looks wrong. See [this comment](https://github.com/golang/go/issues/7819#issuecomment-66093841)  about a similar issue to get a flavour...

Comment (trisberg) [reply cmd/commands/shared.go]: go figure :) thanks for the link

**Comment by glyn** [comment]
Merged as 15993c36573334edf62d5207352a9999cd4dd592.

**Pull request closed without merging**